src/loaders/OmeZarrLoader.ts

```diff
@@ -385,6 +392,18 @@ class OMEZarrLoader extends ThreadableVolumeLoader {
       });
     });
 
+    const alldims: VolumeDims[] = [];
+    source0.scaleLevels.map((level, i) => {
+      const dims = new VolumeDims();
+
+      dims.spaceUnit = spatialUnit;
+      dims.timeUnit = timeUnit;
+      dims.shape = this.orderByTCZYX(level.shape, 1);
+      dims.spacing = this.getScale(i);
+      dims.dataType = level.dtype;
+
+      alldims.push(dims);
+    });
```

Comment:
Might be missing something but if this is a `map`, wouldn't this work?

```suggestion
    const alldims = source0.scaleLevels.map((level, i) => {
      const dims = new VolumeDims();

      dims.spaceUnit = spatialUnit;
      dims.timeUnit = timeUnit;
      dims.shape = this.orderByTCZYX(level.shape, 1);
      dims.spacing = this.getScale(i);
      dims.dataType = level.dtype;

      return dims;
    });
```

Reply:
haha yes I have no idea why I didn't do it like that.  will update